pkg/kritis/cryptolib/verifier_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cryptolib
+
+import (
+	"testing"
+)
+
+func TestVerifyAttestation(t *testing.T) {
+	att := &Attestation{
+		PublicKeyID:       "key-id",
+		Signature:         []byte("signature"),
+		SerializedPayload: []byte("payload"),
+	}
+	matchingKey := NewPublicKey(Pkix, []byte("key-data"), "key-id")
```

Comment:
Should test coverage here also includes other key types?

Reply:
It probably should. To be clear, all the key-type specific verification tests will be happening in other test files (e.g. pgp_test.go). The tests here should only test high-level behavior of the `VerifyAttestation` method (i.e., key lookup, routing to the correct verify method).

For this test, I only included tests for key lookup. Does it also make sense to add tests for routing to the correct verify methods in this PR?

Reply:
That code is simple enough and I don't see an elegant way to test it so I think it's fine as is